src/Cli.Tests/EndToEndTests.cs

```diff
@@ -296,6 +296,34 @@ public void TestAddTelemetry(string? appInsightsEnabled, string appInsightsConnS
         Assert.AreEqual("InstrumentationKey=00000000", updatedConfig.Runtime.Telemetry.ApplicationInsights.ConnectionString);
     }
 
+    /// <summary>
+    /// Test to verify sucess or failure based on different values for
+    /// depth limit in GraphQL runtime settings.
```

Comment:
Needs more specificity: this test checks behavior of executing `dab runtime --query-depth {value}`